content/guides/admin-set-up/finalize-plans-and-setup.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Finalize plans and begin setup
+description: Collaborate with your MDM team to distribute configurations and set up SSO and Docker product trials.
+weight: 20
+---
+
+## Step one: Send finalized settings files to the MDM team 
+
+After reaching an agreement with the relevant teams on your baseline and security configurations as outlined in module one, follow the instructions in the [Settings Management](/manuals/security/for-admins/hardened-desktop/settings-management/_index.md) documentation to create the `admin-settings.json` file that captures these configurations.
+
+Once the file is ready, collaborate with your MDM team to deploy the `admin-settings.json` file, along with your chosen method for [enforcing sign-in](/manuals/security/for-admins/enforce-sign-in/_index.md)..
+
+> [!IMPORTANT]
+>
+> It’s highly recommended that you test this first with a small number of Docker Desktop developers to verify the functionality works as expected before deploying more widely.
+
+## Step two: Manage your organizations
+
+If you have more than one organization, it’s recommended that you either consolidate them into one organization or create a [Docker company](/manuals/admin/company/_index.md) to manage multiple organizations. Work with the Docker Customer Success and Implementation teams to make this happen.
```

Comment:
Not a blocker, this is more-so for my learning: When we instruct a user to reach out to or work with internal Docker teams, do we typically provide a contact email?